target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -415,6 +415,9 @@ ramips_board_detect() {
 	*"R6220")
 		name="r6220"
 		;;
+	*"RAKwireless RAK633")
```

Comment:
@eikepedia You can drop this as it's preferred to use board name from DTS compatible.